app/apps/instrumented_pausetimes_parallel.py

```diff
@@ -1,27 +1,21 @@
-import streamlit as st
```

Comment:
The runtime is no longer instrumented. Best to call it `pausetimes_parallel.py`. Similarly for the sequential one.